src/scripts/app.js

```diff
@@ -0,0 +1,3 @@
+import Alpine from "alpinejs"
```

Comment:
```suggestion
import Alpine from "alpinejs"
```
`package.json` 沒看到有 "alpinejs"，這裡有用到嗎？